packages/react-ui-core/src/Ratings/SampleSvgs/Circle.js

```diff
@@ -0,0 +1,47 @@
+import React, { Component } from 'react'
```

Comment:
There is no reason to name this directory `SampleSvgs`.  Please either move `Circle.js` out or rename the directory accordingly.